modules/EvseManager/evse/evse_managerImpl.cpp

```diff
@@ -137,6 +137,15 @@ void evse_managerImpl::ready() {
 
             auto reason = mod->charger->getSessionStartedReason();
 
+            if (mod->config.disable_authentication && reason == types::evse_manager::StartSessionReason::EVConnected) {
+                // Free service, authorize immediately
+                types::authorization::ProvidedIdToken provided_token;
+                provided_token.authorization_type = types::authorization::AuthorizationType::RFID;
+                provided_token.id_token = "FREESERVICE";
```

Comment:
Adding
```cpp
provided_token.prevalidated = true;
```
here might make the intention even clearer

Reply:
added